src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
         compute! = (out, Y, p, t) ->
-            compute_photosynthesis_net_leaf!(out, Y, p, t, land_model),
+            compute_root_area_index!(out, Y, p, t, land_model),
+    )
+
+        # Stem area index
+    add_diagnostic_variable!(
+        short_name = "sai",
+        long_name = "Stem area Index",
+        standard_name = "stem_area_index",
+        units = "m^2 m^-2",
+        comments = "The area index of stems, expressed in surface area of stems per surface area of ground.",
+        compute! = (out, Y, p, t) ->
+            compute_stem_area_index!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Photosynthesis
+	    # GPP - Gross Primary Productivity
     add_diagnostic_variable!(
         short_name = "gpp",
         long_name = "Gross Primary Productivity",
         standard_name = "gross_primary_productivity",
-        units = "",
-        comments = "Net photosynthesis of the canopy.",
+        units = "mol CO2 m^-2 s^-1",
+        comments = "Net photosynthesis (carbon assimilation) of the canopy. This is equivalent to leaf net assimilation scaled to the canopy level.",
         compute! = (out, Y, p, t) ->
             compute_photosynthesis_net_canopy!(out, Y, p, t, land_model),
     )
+	
+        # Leaf net photosynthesis
+    add_diagnostic_variable!(
+        short_name = "an",
+        long_name = "Leaf Net Photosynthesis",
+        standard_name = "leaf_net_photosynthesis",
+        units = "mol CO2 m^-2 s^-1", # soil CO2 microbial source was in kg co2. we should make a choice and be constant throughout
+        comments = "Net photosynthesis (carbon assimilation) of a leaf, computed for example by the Farquhar model.",
+        compute! = (out, Y, p, t) ->
+            compute_photosynthesis_net_leaf!(out, Y, p, t, land_model),
+    )
 
+        # Leaf respiration
     add_diagnostic_variable!(
         short_name = "rd",
         long_name = "Leaf Respiration",
         standard_name = "leaf_dark_respiration",
-        units = "",
+        units = "mol CO2 m^-2 s^-1",
         comments = "Leaf respiration, called dark respiration because usually measured in the abscence of radiation.",
         compute! = (out, Y, p, t) ->
             compute_respiration_leaf!(out, Y, p, t, land_model),
     )
 
+        # Vcmax25
     add_diagnostic_variable!(
         short_name = "vcmax25",
         long_name = "Vcmax25",
         standard_name = "vcmax25",
-        units = "",
+        units = "mol CO2 m^-2 s^-1",
         comments = "The parameter vcmax at 25 degree celsius. Important for the Farquhar model of leaf photosynthesis.",
         compute! = (out, Y, p, t) -> compute_vcmax25!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Radiative Transfer
+	    # NIR - near infrared radiaton
+    add_diagnostic_variable!(
+        short_name = "nir",
+        long_name = "Near Infrared Radiation",
+        standard_name = "near_infrared_radiation",
+        units = "W m^-2",
```

Comment:
I think NIR, ANIR, TNIR, RNIR and same for Par are:
mol photons per m^2 per 
second for a radiation band.

Reply:
We mention this before, but to be correct: 
PAR (radiation) should be `W m^-2`
PPFD (photons flux density) should be `mol photons m^2 s^-1`
I don't really mind and I understand we can name something and chose its units, but I've been asked by reviewers to correct this before (so it might happen again).

Reply:
(I updated to mol photons m^-2 s^-1 as this is what we have)